test/adapter.spec.js

```diff
@@ -465,7 +465,7 @@ describe('jasmine adapter', function () {
     })
 
     it('should return the all stack entries if every entry is irrelevant', function () {
-      spyOn(window, 'isExternalStackEntry').and.returnValue(true)
+      spyOn(window, 'isExternalStackEntry').and.returnValue(false)
```

Comment:
Please add a comment here as the two cases are confusing (which is why the bug was in the code).  For example:
```
// Check the case where the filteredStack is empty
```

Reply:
Thanks!
I committed at https://github.com/karma-runner/karma-jasmine/pull/206/commits/c25a0b8e8e221368a8a21957e04f6b27aa70b27f